web/src/components/molecules/Content/ContentDetailsMolecule/index.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from "@emotion/styled";
+
+import Content from "@reearth-cms/components/atoms/Content";
+import ContentForm from "@reearth-cms/components/molecules/Content/ContentForm";
+import { ItemField } from "@reearth-cms/components/molecules/Content/types";
+import { Model } from "@reearth-cms/components/molecules/Schema/types";
+
+export type Props = {
+  model?: Model;
+  modelsMenu: React.ReactNode;
+  onItemCreate: (data: { schemaID: string; fields: ItemField[] }) => Promise<void>;
+  onItemUpdate: (data: { itemID: string; fields: ItemField[] }) => Promise<void>;
+  initialFormValues: { [key: string]: any };
+  defaultFormValues: { [key: string]: any };
+  onBack: () => void;
+  itemId?: string;
+};
+
+const ContentDetailsMolecule: React.FC<Props> = ({
+  model,
+  modelsMenu: ModelsMenu,
+  onItemCreate,
+  onItemUpdate,
+  initialFormValues,
+  defaultFormValues,
+  onBack,
+  itemId,
+}) => {
+  return (
+    <PaddedContent>
+      <LeftPaneWrapper>{ModelsMenu}</LeftPaneWrapper>
+      <ContentChild>
+        <ContentForm
+          onBack={onBack}
+          itemId={itemId}
+          onItemCreate={onItemCreate}
+          onItemUpdate={onItemUpdate}
+          model={model}
+          defaultFormValues={defaultFormValues}
+          initialFormValues={initialFormValues}
+        />
+      </ContentChild>
+    </PaddedContent>
```

Comment:
You repeat this style in many places. Please make a generic component and use that instead of copy/pasting a bunch.

Reply:
@KaWaite  I made a generic component `ContentWrapper` which contain these parts (PaddedContent wrapping a LeftPaneWrapper, ContentChild).